cmd/minishift/cmd/config/config.go

```diff
@@ -77,6 +77,9 @@ var (
 	OpenshiftEnv      = createFlag("openshift-env", nil, nil, nil, false)
 	Metrics           = createFlag("metrics", SetBool, nil, nil, true)
 	Logging           = createFlag("logging", SetBool, nil, nil, true)
+	// future enabled flags
+	ServiceCatalog      = createFlag("service-catalog", SetBool, nil, nil, true)
+	ExtraClusterUpFlags = createFlag("extra-clusterup-flags", SetString, nil, nil, true)
```

Comment:
See comment below, The idea, in the end, is to allow a command which is currently experimental, to be promoted to an actual command. In the case of `service-catalog`, we are considering to include it as a default feature. While the `extra-clusterup-flags` is to accommodate future experimental flags to be tested early by the development and evangelists team.